frontend/pages/SoftwarePage/SoftwareOSDetailsPage/SoftwareOSDetailsPage.tsx

```diff
@@ -103,7 +102,13 @@ const SoftwareOSDetailsPage = ({
     {
       enabled: !!osVersionIdFromURL,
       select: (data) => data.os_version,
-      onError: (error) => handlePageError(error),
+      onError: (error) => {
+        // 403s returned for both non-existent and non-accessable entities
```

Comment:
typo *non-accessible

Reply:
I knew something didn't look right there!

Reply:
100 ty for jumping on error states